styles.css

```diff
@@ -1,16 +1,23 @@
 body {
-  display: flex;
+  display: grid;
   justify-content: center;
+  margin: 0;
+  padding: 0;
 }
 
 .main-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: 10;
+  align-items: center;
   width: 375px;
-  display: flex;
-  flex-direction: column;
+  margin: 0;
+  border: 2px solid red;
 }
```

Comment:
- [ ] There is still white space on the right and left sides of your page. The page is not also optimized for the mobile version well. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version to make it responsive. :+1: 

- [ ] Kindly remove the red border from your page. :+1: 

https://user-images.githubusercontent.com/70483835/164817852-25bc3e91-7e0f-407f-9235-177b44b86955.mp4

Reply:
Red border removed.
Problem with whitespaces and responsiveness I think is solved, but more corrections are needed please let me know.
Readme.md file was improved.
Thank you very much for your kind feedback. I am learning a lot.